### PR TITLE
add warning for template option

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by Mashimiao** [comment]
ping @opencontainers/runtime-tools-maintainers

File: man/oci-runtime-tool-generate.1.md [hunk 1]
```diff
@@ -317,6 +317,7 @@ read the configuration from `config.json`.
 **--template**=PATH
   Override the default template with your own.
   Additional options will only adjust the relevant portions of your template.
+  whether template is valid or not will not be checked, this may lead to finally generated config file can't work directly. Currently, users should guarantee template is valid themselves.
```
Comment (mrunalp) [inline man/oci-runtime-tool-generate.1.md]: what do you mean by can't work directly?

Comment (Mashimiao) [inline man/oci-runtime-tool-generate.1.md]: For example, as we don't have template validation now, if there are duplicated namespace entries in user's template, the final config file will also have duplicated namespace. Then the config file is invalid and can't be used directly. 
We need to solve such problems, but not easy, should reform and reuse validate fucntions, currently adding warning will be better.

Comment (mrunalp) [inline man/oci-runtime-tool-generate.1.md]: @Mashimiao Thanks! Maybe word this as
"Templates are not validated for correctness, so the user should ensure that they are correct."

Comment (Mashimiao) [inline man/oci-runtime-tool-generate.1.md]: fine, updated

**Comment by Mashimiao** [comment]
@opencontainers/runtime-tools-maintainers  PTAL

**Comment by liangchenye** [comment]
@Mashimiao I think we should add 'template validation' .

**Comment by mrunalp** [comment]
We can add that in a follow-on I think. LGTM.

**Comment by Mashimiao** [comment]
@liangchenye If we want to add 'template validation', we need to make some modifications on Generation and validaiton. I think we can't finish them in short period, so I submitted this PR to give a warn to users.
Agree with @mrunalp,  we can add it in another PR in the future.

**Comment by liangchenye** [comment]
OK, I add an issue to keep track of this.  https://github.com/opencontainers/runtime-tools/issues/403

**Comment by liangchenye** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**